cpp/ql/src/Critical/FlowAfterFree.qll

```diff
@@ -0,0 +1,95 @@
+import cpp
+import semmle.code.cpp.dataflow.new.DataFlow
+
+/** Signature for a predicate that holds for a dataflow node and an expression whichs holds if the sink is the sought target */
+signature predicate flowTo(DataFlow::Node dfe, Expr e);
```

Comment:
I think this should be called `sink`. `flowsTo` is very confusing when one of the internal predicates in this module is also called `flowsTo`.